crates/ide/src/syntax_highlighting/tags.rs

```diff
@@ -95,7 +117,17 @@ impl HlTag {
             HlTag::EscapeSequence => "escape_sequence",
             HlTag::FormatSpecifier => "format_specifier",
             HlTag::Keyword => "keyword",
-            HlTag::Punctuation => "punctuation",
+            HlTag::Punctuation(punct) => match punct {
+                HlPunct::Bracket => "bracket",
+                HlPunct::Brace => "brace",
+                HlPunct::Parenthesis => "parentheses",
```

Comment:
Hmm.

This is the plural, I think.

Reply:
I always mix those up

Reply:
Oof, how do I run the HTML highlighting tests? Even with `RUN_SLOW_TEST=1 cargo test` nothing fails after changing that line.

Reply:
Never mind, saving the changed files is a good idea :smile:.